src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_read_obs.F90

```diff
@@ -4584,6 +4585,534 @@ subroutine read_obs_SMOS( date_time, N_catd, this_obs_param,      &
     
   end subroutine read_obs_SMOS
 
+
+  ! *****************************************************************
+  
+  subroutine read_MODISscf_hdf( N_files, date_time, N_data, fnames, &
+       MODIS_lon, MODIS_lat, MODIS_SCF)
+    
+    ! Purpose: read preprocessed (renamed) snow cover data from daily MODIS Terra MOD10C1, version 6.1 (https://nsidc.org/data/mod10c1/versions/61) 
+    !   - Data currently located at /discover/nobackup/projects/S2SHMA/MODIS/MOD10C1_V61/ (2010-2022)
```

Comment:
Not sure what "preprocessed" means here.   The files in the Discover dir appear to be the original MOD10C1.*.061.* files except for the removal of the portion of the file name that  reflects the granule production date.   Might need a process like that for SMAP Tb obs, whereby a text file with a list of file names is generated outside of GEOSldas and then read into GEOSldas.

Reply:
Just the renaming, but I think we will need to include a filelist because there is a period in Feb 2016 where there are no modis data